anyway/models.py

```diff
@@ -167,14 +182,23 @@ def get_id(self):
         return self.id
 
 
-class Role(Base, RoleMixin):
-    __tablename__ = "roles"
-    id = Column(Integer(), primary_key=True)
-    name = Column(String(80), unique=True)
+class Roles2(Base):
+    __tablename__ = "roles2"
```

Comment:
If we are changing the implementation of `Roles`, Is it possible to stay with the same name? `Roles2` is not a good name.

Reply:
The original tables are still in the DB, i don't know what data there is in there - so i prefered to create a new tables,
This way has another advantage - if we roll back the code to an older revision the code will still work with the old data.

I have no problem with changing the table name - what do think about `NewRoles`?

Reply:
I think new is better than Roles2. Another option is to rename the current table to old_..., Your call.

Reply:
@BarVolunteering we don't need the old tables / data in these tables - let's both delete old code, remove old tables and create new ones with same indicative names: Users, Roles, etc

Reply:
@atalyaalon Are you sure about deleting the old tables? - i can change their names

Reply:
@BarVolunteering yes I'm sure
Correct me if I'm wrong - we're not using them anywhere in the code right now - hence there is no need for them - you can delete them.

Reply:
We are not using them, I will delete them.